head_test.go

```diff
@@ -34,7 +34,11 @@ import (
 )
 
 func BenchmarkCreateSeries(b *testing.B) {
-	lbls, err := labels.ReadLabels(filepath.Join("testdata", "20kseries.json"), b.N)
+	n := b.N
+	if n > 20000 {
+		n = 20000
+	}
+	lbls, err := labels.ReadLabels(filepath.Join("testdata", "20kseries.json"), n)
```

Comment:
can we use `genSeries` here?

Reply:
We may need some options for `genSeries` in that case. For example `20kseries.json` has changing label names for series (label names do repeat), but `genSeries` will have the same label names for all the series.